website/content/docs/connect/proxies/envoy-extensions/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: docs
+page_title: Envoy Extensions
+description: >-
+  Learn how Envoy extensions enables you to add support for additional Envoy features without modifying the Consul codebase. 
+---
+
+# Envoy extensions
+
+This topic provides an overview of Envoy extensions in Consul service mesh deployments. You can modify Consul-generated Envoy resources to add additional functionality without modifying the Consul codebase.
+
+## Introduction
+
+Historically, Consul supported modifying the Envoy resources it generates through [escape hatches](/consul/docs/connect/proxies/envoy#escape-hatch-overrides). However, implementing escape hatches requires rewriting the Envoy resources so that they are compatible with Consul, a task that also requires understanding how Consul names Envoy resources and enforces intentions.
```

Comment:
```suggestion
Consul supports two methods for modifying Envoy behavior. You can either modify the Envoy resources Consul generates through [escape hatches](/consul/docs/connect/proxies/envoy#escape-hatch-overrides) or configure your services to use Envoy extensions using the `EnvoyExtension` parameter. Implementing escape hatches requires rewriting the Envoy resources so that they are compatible with Consul, a task that also requires understanding how Consul names Envoy resources and enforces intentions.
```
Trying to tweak this so that we are creating timeless docs, as opposed to setting the information up as "this is the old way and this is the new way" per our style guide.